factor_analyzer/factor_analyzer.py

```diff
@@ -19,35 +19,135 @@
 from sklearn.linear_model import LinearRegression
 
 
-def read_file(file_path):
```

Comment:
Why did you get rid of this function?

Reply:
It seemed a bit superfluous in this package, especially since we already have the more robust `read_from_file()` in `rsmtool`.

I don't have extremely strong feelings about it, if you think we should leave it.